packages/iov-cli/src/tsrepl.ts

```diff
@@ -1,7 +1,7 @@
 import { diffLines } from "diff";
 import { join } from "path";
 import { Recoverable, REPLServer, start } from "repl";
-import { register, Register, TSError } from "ts-node";
+import { Register, register, TSError } from "ts-node";
```

Comment:
🤔

Reply:
That's the JS default sort order:

```
> ["register", "Register"].sort()
[ 'Register', 'register' ]
```

No idea why tslint wanted something different before. Maybe tslint allowed both?